Add data about customer workloads in `Pod request vs usage`.

whites11 opened the pull request:
This pr adds 2 new panels to the `Pod request vs usage` dashboard.
The new panels show data about difference between memory request and limits in all pods (including customer workloads).

![customer](https://user-images.githubusercontent.com/868430/189859860-f3e618f6-1cdb-44a7-8068-eb58027bf0b8.png)


### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

hervenicol commented:
What is the difference between both graphs?
If I understand it right, one is %, the other one is MB.
I don't think there's much value in having the evolution over time. Wouldn't it fit all in a table, with instant values?

whites11 commented:
> What is the difference between both graphs? If I understand it right, one is %, the other one is MB.

Correct, one one is relative, the other is absolute value. Both are important IMHO

> I don't think there's much value in having the evolution over time. Wouldn't it fit all in a table, with instant values?

I disagree. If you have an overloaded node during the night, you want to know what was running at that point in time.
Think about night batch jobs.

hervenicol commented:
> If you have an overloaded node during the night, you want to know what was running at that point in time.
> Think about night batch jobs.

Good point.
However, maybe querying last data, and having a "last seen" column would do the job?
For a specific pod / application, the value is not supposed to change over time. That's why I'm a bit uncomfortable with the graphs, I feel like it's not the best way to view this data, and I'm wondering if there would be a better way.

Also, would be nice to have a detailed description per graph, with this kind of info ("we can also view data for pods that are not here anymore"), so next time someone wants to "optimize" the dashboard he understands the whereabouts.

hervenicol reviewed (approved):
My concerns are just ideas for improvement, not a reason to block ;)

TheoBrigitte reviewed (commented):
Nice job, but I follow hervé here. I would appreciate a details description and a column with instant values.

whites11 commented:
There is no clear consensus about this change, we don't own the dashboard and I don't have budget to invest more time on addressing the comments. Closing
